pastis/config_pastis.ini

```diff
@@ -151,4 +156,13 @@ second_astigm0: 12
 second_astigm: 13
 
 [calibration]
-local_zernike = 1
\ No newline at end of file
+local_zernike = 1
+single_aberration = 1.
+max_LO = 15
+max_MID = 6
+max_HI = 24
+num_DM_act = 4
+
+[e2eparameters]
+c_target_log = -11
+n_repeat = 20
```

Comment:
The parameters in this section should not be read from the configfile becuase they will be very different for the different instruments and used cases. Please delete.

Reply:
@ananyasahoo1904 here

Reply:
These paremeters are needed for Luvoir, so I added them under [LUVOIR]